docs/userguide.md

```diff
@@ -18,26 +28,17 @@ export PATH=$(pwd)/bin:${PATH}
 
 Or you can install them manually yourself using the guidelines provided below.
 
-#### apiserver-builder
-
-This repo depends on
-[apiserver-builder](https://github.com/kubernetes-incubator/apiserver-builder)
-to generate code and build binaries. Download a [recent
-release](https://github.com/kubernetes-incubator/apiserver-builder/releases)
-and install it in your `PATH`.
-
-#### cluster-registry
-
-Get version 0.0.4 of the [Cluster
-Registry](https://github.com/kubernetes/cluster-registry/releases/tag/v0.0.4)
-and install it in your `PATH`.
-
 #### kubectl
 
 Download a version of
 [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and
 install it in your `PATH`.
 
+#### kubebuilder
+
+Get the latest version of [`kubebuilder`](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/quick_start.md#installation)
```

Comment:
perfer use `0.1.11 or above` as kubebuilder version.

Reply:
The latest is 0.1.12, which I understand is required, so I think either latest or >= 0.1.12 works.

Reply:
+1 for this